src/test-fn/kotlin/com/anaplan/engineering/vdmgradleplugin/TypeCheckTest.kt

```diff
@@ -73,14 +72,12 @@ class TypeCheckTest(
                 projectDir = dir,
                 tasks = arrayOf(task),
                 fail = !expectSuccess)
-        val libFile = File(dir, "build/vdm/generated.lib")
-        Assert.assertEquals(expectSuccess, libFile.exists())
 
-        if (expectSuccess) {
-            val vdmsl = VDMSL()
-            vdmsl.parse(listOf(libFile))
-            checkModules(vdmsl.interpreter.modules)
-        }
+//        if (expectSuccess) {
+//            val vdmsl = VDMSL()
+//            vdmsl.parse(listOf(libFile))
+//            checkModules(vdmsl.interpreter.modules)
+//        }
```

Comment:
Without saving the log of the type checking task, I don't think there is a way to know how many modules were found. I left the comment in for now because outright removing this part feels like removing a large chunk of the test.

Reply:
this should probably check the status file

checking the log may be a bit too much parsing effor